products/sle/main.pm

```diff
@@ -572,6 +572,7 @@ sub load_consoletests() {
         loadtest "console/vim.pm";
         if (!is_staging()) {
             loadtest "console/firewall_enabled.pm";
+            loadtest "console/zypper_lifecycle.pm";
```

Comment:
ah, ok. should be >=12-SP2 as it's a new feature for that version but as it's a feature validation test I understood it should not just be part of extratests.